mysql/datadog_checks/mysql/mysql.py

```diff
@@ -423,6 +423,10 @@ def _collect_metrics(self, db, tags):
             and above_560
             and self.performance_schema_enabled
         ):
+            self.warning(
+                "[Deprecated] The `extra_performance_metrics` option will be removed in a future release. "
+                "Utilize the `custom_queries` feature if the functionality is needed.",
+            )
```

Comment:
should we remove it from the docs here https://github.com/DataDog/integrations-core/blob/master/mysql/datadog_checks/mysql/data/conf.yaml.example#L307?

Reply:
@jmeunier28 yeah, a follow up will be made for that